eras/alonzo/impl/src/Cardano/Ledger/Alonzo/Rules/Bbody.hs

```diff
@@ -209,14 +209,19 @@ alonzoBbodyTransition =
           == fromIntegral (bhviewBSize bh)
             ?! injectFailure
               ( ShelleyInAlonzoBbodyPredFailure
-                  (WrongBlockBodySizeBBODY actualBodySize (fromIntegral $ bhviewBSize bh))
+                  ( WrongBlockBodySizeBBODY $
+                      Mismatch
+                        { mismatchSupplied = actualBodySize
+                        , mismatchExpected = fromIntegral $ bhviewBSize bh
+                        }
+                  )
               )
 
         actualBodyHash
           == bhviewBHash bh
             ?! injectFailure
               ( ShelleyInAlonzoBbodyPredFailure
-                  (InvalidBodyHashBBODY @era actualBodyHash (bhviewBHash bh))
+                  (InvalidBodyHashBBODY @era $ Mismatch (bhviewBHash bh) actualBodyHash)
```

Comment:
Looks like you did apply my suggestion [from previous review](https://github.com/IntersectMBO/cardano-ledger/pull/4649#discussion_r1779503333) for Shelley but missed applying it here in Alonzo:
```suggestion
                  InvalidBodyHashBBODY $ Mismatch
                        { mismatchSupplied = actualBodyHash
                        , mismatchExpected = bhviewBHash bh
                        }
                 )
```

Reply:
Thanks @lehins. I don't know how I missed this one. :+1: